js/src/lib/components/InferenceWidget/shared/helpers.ts

```diff
@@ -180,16 +180,14 @@ export async function getResponse<T>(
 }
 
 
-export async function getModelLoadInfo(url: string, repoId: string, includeCredentials = false): Promise<ModelLoadInfo> {
+export async function getModelLoadInfo(url: string, repoId: string, includeCredentials = false): Promise<ModelLoadInfo | undefined> {
 	const response = await fetch(`${url}/status/${repoId}`, {credentials: includeCredentials ? "include" : "same-origin"});
 	const output = await response.json();
-	if (response.ok && typeof output === 'object' && output.loaded !== undefined) {
-		const status = output.loaded ? 'loaded' : 'unknown';
-		const compute_type = output.compute_type;
-		return {status, compute_type}
+	if (response.ok) {
+		return output as ModelLoadInfo;
 	} else {
 		console.warn(response.status, output.error);
-		return {status: 'error'};
```

Comment:
@aliabid94 could you explain removal of this line? The error is handled somewhere?

Reply:
I'm using undefined to mean an unknown model status. either we get a ModelLoadInfo object if we communicated with the server or undefined if not.

Reply:
e.g. here: https://github.com/huggingface/hub-docs/pull/895/files#diff-47bdcb4d33fde7883ff228c78a96272ed67aa281dfcbceef721e9a80fcb8a211R28